.github/workflows/ci.yaml

```diff
@@ -3,17 +3,63 @@ name: default
   push:
     branches:
       - main
+      - release-*
     tags:
       - v*
-  pull_request: {}
-env:
-  CI_ARGS: --cache-from=type=registry,ref=registry.dev.siderolabs.io/${GITHUB_REPOSITORY}:buildcache --cache-to=type=registry,ref=registry.dev.siderolabs.io/${GITHUB_REPOSITORY}:buildcache,mode=max
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+    branches:
+      - main
+      - release-*
+  schedule:
```

Comment:
what does the schedule mean?

Reply:
I mean not sure which step runs there

Reply:
I see that reproducibility depends on label, not on schedule event?

Reply:
ahh missed it